src/main/java/onboarding/Problem3.java

```diff
@@ -1,8 +1,11 @@
 package onboarding;
 
+import problem3.ThreeSixNineGame;
+
 public class Problem3 {
+
     public static int solution(int number) {
-        int answer = 0;
-        return answer;
+        int result = ThreeSixNineGame.playGame(number);
+        return result;
```

Comment:
result 변수를 선언하지 않고 playGame(number) 함수를 바로 return 하는 것은 어떨까요?
```suggestion
return ThreeSixNineGame.playGame(number);
```
코드가 좀 더 깔끔해질 것 같다는 제 생각입니다...!